contracts/Staking.sol

```diff
@@ -462,6 +478,10 @@ contract Staking is IStaking, Initializable, OwnableUpgradeable, Constants {
             if (time < lockPeriod) {
                 break;
             }
+            if (time == block.timestamp) {
```

Comment:
[Question] Why need this check?

Reply:
when we widthdraw, we need to skip the canceled unbond request.
Since delegator cancel an unbond request, we delete that unbond request
 `delete unbondingAmount[msg.sender][unbondReqId];`
so the `unbondingAmount.startTime` will be set to zero, 
and in this case, `time = block.timestamp - unbondingAmount[msg.sender][i].startTime;`
we can know time = block.timestamp is canceled unbond request.

Reply:
Better add a comment for this line to explain the logic

Reply:
Use amount other than time to check the condition